app/pods/components/web-components/au-dropdown/item/component.js

```diff
@@ -1,6 +1,12 @@
 import Component from '@glimmer/component';
 
 export default class Item extends Component {
+  get class() {
+    if (this.args.class) {
+      return this.args.class;
+    }
+    return '';
```

Comment:
Hier willen we `null` hebben staan.. een lege quote zetten, genereerde een spatie op de plekken waar de css staat in de html geloof ik.. dat willen we liever niet.